lighthouse-core/report/scripts/perf-x.js

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
does it make sense to move this under `lighthouse-cli/performance-experiment/` to keep everything in one place?